server/models/Template.model.js

```diff
@@ -10,7 +10,10 @@ const TemplateSchema = new Schema({
   // title of the document
   title: { type: String, required: true },
 
-  // string for use in template rendering
+  // Name of the template file
+  fileName: { type: String, required: true },
+
+  // base64-encoded byte string
   template: { type: String, required: true },
```

Comment:
I would rather this be a `Buffer` to recognize it is blob storage. Base64 is wasting almost half of the space.